scripts/changelog.sh

```diff
@@ -150,33 +150,43 @@ function nextminor {
     NEXT_VERSION=$(npx -y changie@$CHANGIE_VERSION next minor)
     # Remove all existing per-release changelogs
     rm ./.changes/*.*.*.md
-    # Remove all unreleased changes
-    rm ./.changes/unreleased/*.yaml
-    # Remove all backported changes
-    rm ./.changes/backported/*.yaml
+    
     # Create a new empty version file for the next minor version
     touch ./.changes/$NEXT_VERSION.md
     
+    LATEST_MAJOR_MINOR=$(echo $LATEST_VERSION | awk -F. '{print $1"."$2}')
+    NEXT_MAJOR_MINOR=$(echo $NEXT_VERSION | awk -F. '{print $1"."$2}')
+    
+    # Create a new changes directory for the next minor version
+    mkdir ./.changes/v$NEXT_MAJOR_MINOR
+    touch ./.changes/v$NEXT_MAJOR_MINOR/.gitkeep
+
```

Comment:
idea/nit: as well as making the new changes folder, could this function also clear any stale change files for past versions that are still on the main branch?